tvtk/pyface/picker.py

```diff
@@ -81,7 +83,7 @@ class PickHandler(HasTraits):
     what you need.  Each time a pick occurs the handle_pick is called
     by the `Picker` class."""
 
-    def handle_pick(self, data):
+    def handle_pick(self, data, renwin):
```

Comment:
I do not like that the API has changed suddenly, if anyone has implemented a custom `PickHandler`, it will fail.

Reply:
If you want to pass additional data, you can add some extra traits to the `PickedData` class and add the renwin and text actor which can be optional.